src/components/Header/Header.tsx

```diff
@@ -34,11 +34,13 @@ const useStyles = makeStyles<Theme>(theme => ({
   paperAppBarBackground: {
     background: `linear-gradient(${theme.palette.background.default}, ${theme.palette.background.paper})`,
   },
-  loaderShadow: {
-    boxShadow: theme.shadows[2],
+  roundedAppBarBackground: {
```

Comment:
Why Header component implementation needs to be changed in this US?
Noticed a bug, the header is now disappears from the screen at certain time on scroll.

Reply:
Agree, that's not the best place, just was a quick fix of this 
![Screenshot 2019-12-17 at 15 09 45](https://user-images.githubusercontent.com/5225448/71002330-525f6200-20df-11ea-8d0d-dc4dce63e6dc.png)